app/javascript/react/screens/App/Plan/PlanReducer.js

```diff
@@ -35,102 +36,88 @@ const excludeDownloadDoneTaskId = (allDownloadLogInProgressTaskIds, taskId) =>
 const includeDownloadInProgressTaskId = (allDownloadLogInProgressTaskIds, taskId) =>
   allDownloadLogInProgressTaskIds ? allDownloadLogInProgressTaskIds.concat(taskId) : [taskId];
 
-const _formatPlanRequestDetails = data => {
+const processVMTasks = vmTasks => {
   const tasks = [];
-  if (data.miq_request_tasks && data.miq_request_tasks.length) {
-    data.miq_request_tasks.forEach(task => {
-      const taskDetails = {
-        id: task.id,
-        message: task.message,
-        transformation_host_name: task.options && task.options.transformation_host_name,
-        delivered_on: new Date(task.options.delivered_on),
-        updated_on: new Date(task.updated_on),
-        completed: task.message === 'VM Transformations completed' || task.message === 'VM Transformations failed',
-        state: task.state,
-        status: task.status,
-        options: {}
-      };
-
-      taskDetails.options.progress = task.options.progress;
-      taskDetails.options.virtv2v_wrapper = task.options.virtv2v_wrapper;
-
-      if (!task.diskSpaceCompletedGb) {
-        taskDetails.diskSpaceCompletedGb = '0';
-      }
+  vmTasks.forEach(task => {
+    const taskDetails = {
+      id: task.id,
+      message: task.message,
+      transformation_host_name: task.options && task.options.transformation_host_name,
+      delivered_on: new Date(task.options.delivered_on),
+      updated_on: new Date(task.updated_on),
+      completed: task.message === 'VM Transformations completed' || task.message === 'VM Transformations failed',
```

Comment:
Is this a string that might change with translation?

Reply:
Somewhat related: we actually already have an issue about those... if we want to tackle that in a future PR (or in this one): https://github.com/ManageIQ/miq_v2v_ui_plugin/issues/255

"VM Transformations completed" vs "Migrations completed"

Ideally, we can create some sort of dictionary to map them to the UI translation on the Redux end... I don't know if we tackle this now though...

Reply:
Those strings are coming from the backend -- hopefully the backend has gettexted them.

But you are right, this should be -
```js
completed: task.message === __('VM Transformations completed') || task.message === __('VM Transformations failed'),
```

We can't control the backend, but we can at least do our part.

Reply:
I have also found a couple of incorrect `sprintf` usages - i.e. incorrect in the context of i18n.

I'm thinking of handling this string and a bunch of other strings in a separate PR for addressing all known i18n issues.